tabs/src/components/ui/Component.jsx

```diff
@@ -1,11 +1,26 @@
 import React from "react";
+import { Header } from '@fluentui/react-northstar'
+import { Link } from 'react-router-dom'
+import "./Overall.css";
```

Comment:
for best practice, css file name should use lowercase string

Reply:
thanks, I will rename the css file.